source/client/index.js

```diff
@@ -1,7 +1,12 @@
 const React = require('react')
-const withScroll = require('scroll-behavior').default
+const useScroll = require('react-router-scroll/lib/useScroll')
```

Comment:
I'm the same normal, but it's called out directly in the readme, so I assume it would be a breaking change if it does change.

> If you are not using <ScrollContainer>, you can reduce your bundle size by importing the useScroll module directly.

Still happy to change it if you want though?